src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -289,6 +289,22 @@ public InmateDetail courtTransferIn(
         return prisonerReleaseAndTransferService.courtTransferIn(offenderNo, requestForCourtTransferIn);
     }
 
+    @ApiResponses({
```

Comment:
once this is built and ready to test we will remove the old version and this will be renamed back